src/foam/u2/view/UnstyledTableView.js

```diff
@@ -780,7 +780,7 @@ foam.CLASS({
               var clsName  = self.of.name.toLowerCase();
               var propName = axiom.name.toLowerCase();
               return auth.check(ctrl.__subContext__, `${clsName}.column.${propName}`).then(function(enabled) {
-                return enabled && c;
+                return c;
```

Comment:
By removing the enabled check your ignoring the auth/permission check. don't think this is correct

Reply:
Yes, there's no need to perform the check if we're just going to disregard the result.